config/custom-environment-variables.yml

```diff
@@ -1,22 +1,19 @@
-node:
-  config:
-    env: 'NODE_CONFIG_ENV'
+accounts:
+  tokenKey: 'JWT_TOKEN'
 
 rollbar:
   env: 'ROLLBAR_ENV'
   accessToken: 'ROLLBAR_ACCESS_TOKEN'
 
-jwt:
-  token: 'JWT_TOKEN'
```

Comment:
`secret` was a better name imo. Also could we add refresh tokens too into the boiler?